docs: explain how to test single block/superblock (#49725)
freeCodeCamp/freeCodeCamp: docs/how-to-work-on-coding-challenges.md

pnpm run test:curriculum
```

> 2. You can also test a block or a superblock of challenges by passing the environment variable `FCC_BLOCK` or `FCC_SUPERBLOCK` respectively. For example, on 'nix systems:

Author: I meant unix-based systems in general, but we could just say "on Linux".

> 2. You can also test a block or a superblock of challenges by passing the environment variable `FCC_BLOCK` or `FCC_SUPERBLOCK` respectively. For example, on 'nix systems:

Reviewer: ```suggestion
2. To test a block or a superblock of challenges, you can pass the environment variables `FCC_BLOCK` or `FCC_SUPERBLOCK`, respectively. For instance, on Unix-based systems:
```
